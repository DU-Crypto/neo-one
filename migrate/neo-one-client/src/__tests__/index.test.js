/* @flow */
import client, { BasicClient, Client, testClient } from '../index';

test('exports', () => {
  expect(client).toMatchSnapshot();
  expect(testClient).toMatchSnapshot();
  expect(Client).toBeTruthy();
  expect(BasicClient).toBeTruthy();
});
