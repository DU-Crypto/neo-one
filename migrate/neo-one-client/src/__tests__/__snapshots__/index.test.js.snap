// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`exports 1`] = `
Client {
  "_addressVersion": 23,
  "_issueGASFee": "500",
  "_privateKeyVersion": 128,
  "_provider": JSONRPCClientProvider {
    "_provider": JSONRPCHTTPProvider {
      "batcher": DataLoader {
        "_batchLoadFn": [Function],
        "_options": Object {
          "cacheKeyFn": [Function],
          "maxBatchSize": 25,
        },
        "_promiseCache": Array [],
        "_queue": Array [],
      },
    },
  },
  "_utilityTokenHash": "0x602c79718b16e442de58778e148d0b1084e3b2dffd5de6b7b16cee7969282de7",
  "abi": Object {
    "NEP5": [Function],
  },
  "parameters": Object {
    "byType": Object {
      "Array": Object {
        "to": [Function],
        "toNullable": [Function],
      },
      "Boolean": Object {
        "to": [Function],
        "toNullable": [Function],
      },
      "ByteArray": Object {
        "to": [Function],
        "toNullable": [Function],
      },
      "Hash160": Object {
        "to": [Function],
        "toNullable": [Function],
      },
      "Hash256": Object {
        "to": [Function],
        "toNullable": [Function],
      },
      "Integer": Object {
        "to": [Function],
        "toNullable": [Function],
      },
      "InteropInterface": Object {
        "to": [Function],
        "toNullable": [Function],
      },
      "PublicKey": Object {
        "to": [Function],
        "toNullable": [Function],
      },
      "Signature": Object {
        "to": [Function],
        "toNullable": [Function],
      },
      "String": Object {
        "to": [Function],
        "toNullable": [Function],
      },
      "Void": Object {
        "to": [Function],
        "toNullable": [Function],
      },
    },
    "toArray": [Function],
    "toArrayNullable": [Function],
    "toBoolean": [Function],
    "toBooleanNullable": [Function],
    "toByteArray": [Function],
    "toByteArrayNullable": [Function],
    "toHash160": [Function],
    "toHash160Nullable": [Function],
    "toHash256": [Function],
    "toHash256Nullable": [Function],
    "toInteger": [Function],
    "toIntegerNullable": [Function],
    "toInteropInterface": [Function],
    "toPublicKey": [Function],
    "toPublicKeyNullable": [Function],
    "toSignature": [Function],
    "toSignatureNullable": [Function],
    "toString": [Function],
    "toStringNullable": [Function],
    "toVoid": [Function],
  },
}
`;

exports[`exports 2`] = `
Client {
  "_addressVersion": 23,
  "_issueGASFee": "5",
  "_privateKeyVersion": 128,
  "_provider": JSONRPCClientProvider {
    "_provider": JSONRPCHTTPProvider {
      "batcher": DataLoader {
        "_batchLoadFn": [Function],
        "_options": Object {
          "cacheKeyFn": [Function],
          "maxBatchSize": 25,
        },
        "_promiseCache": Array [],
        "_queue": Array [],
      },
    },
  },
  "_utilityTokenHash": "0x602c79718b16e442de58778e148d0b1084e3b2dffd5de6b7b16cee7969282de7",
  "abi": Object {
    "NEP5": [Function],
  },
  "parameters": Object {
    "byType": Object {
      "Array": Object {
        "to": [Function],
        "toNullable": [Function],
      },
      "Boolean": Object {
        "to": [Function],
        "toNullable": [Function],
      },
      "ByteArray": Object {
        "to": [Function],
        "toNullable": [Function],
      },
      "Hash160": Object {
        "to": [Function],
        "toNullable": [Function],
      },
      "Hash256": Object {
        "to": [Function],
        "toNullable": [Function],
      },
      "Integer": Object {
        "to": [Function],
        "toNullable": [Function],
      },
      "InteropInterface": Object {
        "to": [Function],
        "toNullable": [Function],
      },
      "PublicKey": Object {
        "to": [Function],
        "toNullable": [Function],
      },
      "Signature": Object {
        "to": [Function],
        "toNullable": [Function],
      },
      "String": Object {
        "to": [Function],
        "toNullable": [Function],
      },
      "Void": Object {
        "to": [Function],
        "toNullable": [Function],
      },
    },
    "toArray": [Function],
    "toArrayNullable": [Function],
    "toBoolean": [Function],
    "toBooleanNullable": [Function],
    "toByteArray": [Function],
    "toByteArrayNullable": [Function],
    "toHash160": [Function],
    "toHash160Nullable": [Function],
    "toHash256": [Function],
    "toHash256Nullable": [Function],
    "toInteger": [Function],
    "toIntegerNullable": [Function],
    "toInteropInterface": [Function],
    "toPublicKey": [Function],
    "toPublicKeyNullable": [Function],
    "toSignature": [Function],
    "toSignatureNullable": [Function],
    "toString": [Function],
    "toStringNullable": [Function],
    "toVoid": [Function],
  },
}
`;
